packages/file-input/_file.scss

```diff
@@ -1,62 +1,74 @@
 @use "@fremtind/jkl-core/jkl";
 @use "@fremtind/jkl-core/jkl/colors";
 
-@include jkl.light-mode-variables {
-    --jkl-file-color: var(--jkl-color);
-    --jkl-file-thumbnail-background-color: #{jkl.$color-sand};
-    --jkl-file-border-color: #{jkl.$color-dis};
-    --jkl-file-border-color-active: #{jkl.$color-granitt};
-}
-
-@include jkl.dark-mode-variables {
-    --jkl-file-color: var(--jkl-color);
-    --jkl-file-thumbnail-background-color: #{jkl.$color-stein};
-    --jkl-file-border-color: #{jkl.$color-dis};
-    --jkl-file-border-color-active: #{jkl.$color-snohvit};
-}
-
-@include jkl.comfortable-density-variables {
-    --jkl-file-padding: #{jkl.$spacing-16} #{jkl.$spacing-12};
-    --jkl-file-thumbnail-size: #{jkl.rem(64px)};
-    --jkl-file-gap: #{jkl.$spacing-16};
-}
-
-@include jkl.compact-density-variables {
-    --jkl-file-padding: #{jkl.$spacing-12} #{jkl.$spacing-8};
-    --jkl-file-thumbnail-size: #{jkl.rem(44px)};
-    --jkl-file-gap: #{jkl.$spacing-8};
-}
-
 .jkl-file {
     @include jkl.motion("standard", "snappy");
+
     transition-property: background-color, border-color, color;
 
-    color: var(--jkl-file-color);
-    border: 1px var(--jkl-file-border-color) solid;
-    padding: var(--jkl-file-padding);
+    color: var(--jkl-color-border-action);
```

Comment:
Hvis denne blir brukt som tekstfarge er det nok bedre å bruke en av fargerollene for tekst 😊 F.eks. `var(--jkl-color-text-default)`.